packages/retail-ui-extensions/src/components/ScrollView/ScrollView.ts

```diff
@@ -0,0 +1,11 @@
+import {createRemoteComponent} from '@remote-ui/core';
+
+export interface ScrollViewProps {
+  flex?: number;
+  scrollEventThrottle?: number;
+  keyboardShouldPersistTaps?: boolean | 'always' | 'never' | 'handled';
```

Comment:
I'm not sure we should give developers any control over this. What do you think?